scripts/build

```
@@ -27,6 +27,12 @@ else
     git config --global --add safe.directory /go/src/github.com/harvester/harvester
 fi
 
+addons_path=../addons
+if [ ! -d ${addons_path} ];then
+    echo "No existed harvester source. Pulling..."
```

Probably should mention this is "addons":

```suggestion
    echo "No existing harvester addons source. Pulling..."
```